modules/images-other-jenkins-oauth-auth.adoc

```diff
@@ -18,12 +18,28 @@ Jenkins supports both browser and non-browser access.
 
 Valid users are automatically added to the Jenkins authorization matrix at log
 in, where {product-title} `Roles` dictate the specific Jenkins permissions the
-users have.
+users have.  The `Roles` used by default are the `admin`, `edit`, and `view` 
+`Roles` in the `Project` or namespace that Jenkins is running in.
 
 Users with the `admin` role have the traditional Jenkins administrative
 user permissions. Users with the `edit` or `view` role have progressively
 fewer permissions.
 
+Those defaults noted, the which set of {product-title} `Roles` and which Jenkins permissions
+those `Roles` are assigned in the Jenkins instance are configurable.
+
+The login plugin will look for a `ConfigMap` named `openshift-jenkins-login-plugin-config`. Specifically, when running Jenkins in an {product-title} `Pod`, it will look in the namespace that Jenkins is running in. Otherwise, it looks in the namespace specified in the "client ID" as explained in "Secondary Scenarios" down below.
+
+If this plugin finds and can read in that ConfigMap, it then:
+
+ * Treats the key/value pairs in the ConfigMap as Jenkins permission to OpenShift Role mappings.
+ * The key is the Jenkins permission group short ID and the Jenkins permission short ID, with those two separated by a hyphen or - character.
+ * So if you want to add the Overall Jenkins Administer permission to an OpenShift Role, the key should be Overall-Administer
```

Comment:
s/So if/If

Is it Administer or Administrator? 

s/OpenShift Role/{product-title} `Role`

s/Overall-Administer/`Overall-Administer`.

Reply:
Maybe we could talk here about the locale/multi language not supported ?

Reply:
changes done (I think), it is Administer not Administrator in this context ... I'll let you @akram craft the locale change separately once this merges